src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -983,7 +983,9 @@ void goto_instrument_parse_optionst::instrument_goto_program()
        cmdline.isset("custom-bitvector-analysis"))
       config.ansi_c.defines.push_back("__CPROVER_CUSTOM_BITVECTOR_ANALYSIS");
 
-    status() << "Adding CPROVER library" << eom;
+    // add the library
+    status() << "Adding CPROVER library ("
+              << config.ansi_c.arch << ")" << eom;
```

Comment:
This is a very useful change, but I'd appreciate if it could be placed in a separate PR - it's entirely unrelated to the topic of this PR.